packages/xml-views-completion/test/providers/attributeName/prop-event-assoc-spec.ts

```diff
@@ -120,6 +120,62 @@ describe("The ui5-language-assistant xml-views-completion", () => {
         });
       });
 
+      it("will not filter out the current attribute when only attribute name exists", () => {
```

Comment:
This is an **imperative** description of how it does it.
But the **functional** description is more like:
`will suggest attribute which is identical to the user's input prefix`

Reply:
also I did not quite understand `when only attribute name exists`

Reply:
Other tests in this file also mention "filtering out", so I went with a similar description. My tests mention "not filtering out" so maybe it's different... I'll change it

Reply:
I think other tests may mention filter in functional spec manner, e.g: `filter out none public classes` or `filter by prefix` but maybe some other descriptions could also be upgraded.